README.md

```diff
@@ -16,6 +16,9 @@ principles. These were originally designed by Kent Beck, creator of extreme
 programming software development methodology, for Smalltalk's SUnit. It allows
 writing tests, checking results and automated testing in Ruby.
 
+test-unit ships as part of Ruby's standard library. To check which version is included, see https://stdgems.org/test-unit/.
```

Comment:
test-unit isn't bundled as a standard library.
test-unit is bundled as a bundled gem.

See also: https://www.slideshare.net/hsbt/how-to-develop-the-standard-libraries-of-ruby

Reply:
Thanks, how about:
```suggestion
test-unit ships as part of Ruby as a bundled gem. To check which version is included, see https://stdgems.org/test-unit/.
```